docs/sources/write/links/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Links
+description: Understand how to link between pages.
+weight: 600
+aliases:
+  - /docs/writers-toolkit/write/links/
+  - /docs/writers-toolkit/write/references/
+  - /docs/writers-toolkit/writing-guide/references/
+keywords:
+  - Hugo
+  - link
+  - linking
+  - links
+  - ref
+  - references
+  - relref
+---
+
+# Links
+
+Choose your link type based on the applicable scenario:
+
+- [Link from source content that's used in multiple projects](#link-from-source-content-thats-used-in-multiple-projects)
+- [Link to grafana.com pages](#link-to-grafanacom-pages)
+- [Link to external pages](#link-to-external-pages)
+- [Link to page headings](#link-to-page-headings)
+
+Other link types exist in our documentation, but you shouldn't use them:
```

Comment:
I could contrive some alternatives but it's really just `relref` that is discouraged now.

Reply:
Can we say something like, "Although other types of links still function, replace them with ... because it helps with future-oriented compatibility." (Could likely be revised; people generally respond better to positive phrasing.)

Reply:
`
```suggestion
Although these other types of links still function, replace them with full URLs:
```